modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayEnvironmentsAdministrator.testcase

```diff
@@ -71,39 +117,113 @@ definition {
 	}
 
 	@description = "This is a test for LPS-157237 - Verify if as an Administrator I am able to delete Factor Categories"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test AdminCanDeleteCategory {
+		task ("Go to Categories on Manage Sidebar") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			TestrayLayout.clickManageOnSidebar();
 
-		// TODO LPS-157237 AdminCanDeleteCategory pending implementation
+			TestrayManage.selectOptionsManage(itemValue = "Categories");
+		}
+
+		task ("Add a category, Delete it, and check if the category has been deleted") {
+			TestrayManage.addNewCategory(category = "Environments_4 Category");
+
+			TestrayManage.deleteCategory();
 
+			TestrayManage.assertNameNotPresent(name = "Environments_4 Category");
+		}
 	}
 
 	@description = "This is a test for LPS-157307 - Verify if as an Administrator I am able to edit Factor Categories"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test AdminCanEditFactorCategories {
+		task ("Go to Categories on Manage Sidebar") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			TestrayLayout.clickManageOnSidebar();
+
+			TestrayManage.selectOptionsManage(itemValue = "Categories");
+		}
+
+		task ("Create the new category and then edit and state if the name has changed") {
+			TestrayManage.addNewCategory(category = "Environments_6 Category");
 
-		// TODO LPS-157307 AdminCanEditFactorCategories pending implementation
+			TestrayManage.editNameCategory(category = "Environments_6 Category Edited");
 
+			TestrayManage.assertName(name = "Environments_6 Category Edited");
+		}
 	}
 
 	@description = "This is a test for LPS-157326 - Verify if as an Administrator I am able to edit the Option by changing the Category"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test AdminCanEditOptionChangingCategory {
+		task ("Go to Categories on Manage Sidebar and Add Category") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			TestrayLayout.clickManageOnSidebar();
+
+			TestrayManage.selectOptionsManage(itemValue = "Categories");
+
+			TestrayManage.addNewCategory(category = "Environments_8 Category A");
+
+			TestrayManage.addNewCategory(category = "Environments_8 Category B");
+		}
+
+		task ("Go to Options on Manage Sidebar and Add New Option") {
+			TestrayLayout.clickManageOnSidebar();
 
-		// TODO LPS-157326 AdminCanEditOptionChangingCategory pending implementation
+			TestrayManage.selectOptionsManage(itemValue = "Options");
 
+			TestrayManage.createNewOption(
+				categorie = "Environments_8 Category A",
+				key_value = "Environments_8 Option");
+		}
+
+		task ("Check if you can edit the Option by changing the Category") {
+			TestrayManage.editNewOptionName(
+				categorie = "Environments_8 Category B",
+				key_value = "Environments_8 Option");
+
+			TestrayManage.assertNameCategoryInOptions(category = "Environments_8 Category B");
+
+			TestrayManage.assertCategorieNotPresent(key_title = "Environments_8 Category A");
```

Comment:
Hi @joelrocha68 would you please correct this word? "categorie" must be changed to "category"